platform/android/java/src/org/godotengine/godot/Dictionary.java

```diff
@@ -32,35 +32,36 @@
 import java.util.HashMap;
 import java.util.Set;
 
-
 public class Dictionary extends HashMap<String, Object> {
 
 	protected String[] keys_cache;
 
 	public String[] get_keys() {
 
 		String[] ret = new String[size()];
-		int i=0;
+		int i = 0;
 		Set<String> keys = keySet();
 		for (String key : keys) {
 
 			ret[i] = key;
 			i++;
-		};
+		}
+		;
```

Comment:
This should be removed I guess (same for other occurrences below).

Reply:
Sure, I want you to know that this commit contains only linter modifications.
Meaning it's been removed here:
https://github.com/godotengine/godot/pull/16912/files#diff-d98f27dcd89fc8818b4ffd20c1bf1393R48